### DISCUSSION TITLE
2021-09-28 Website Committee

### DISCUSSION THREAD

**Discussion by lauggh** [worldiaday/website#35] [Meetings]
Recording: https://vimeo.com/617095916/4ac20e3a98

- List of users
- The 2018 user research and what needs to be added/researched
- The personas
- The user journeys

**Comment by lauggh** [comment]
@GhadaKandil What other topics do we need to include for our next Website mtg?

**Reply by GhadaKandil** [reply]
I have edited the agenda. All the meeting will revolve around the users, their personas and journeys

**Comment by lauggh** [comment]
Recording added
